WalletWasabi.Backend/wwwroot/index.html

```diff
@@ -378,11 +385,9 @@ <h2 class="text-center text-lowercase mb-4">Frequently Asked Questions</h2>
                     </label>
                     <div class="card-body toggle-content p-0" itemscope itemprop="acceptedAnswer" itemtype="https://schema.org/Answer">
                         <p class="card-text border-top p-2" itemprop="text">
-                            Wasabi generates Bech32 addresses only, also known as bc1 addresses or native SegWit addresses. These addresses start with the characters bc1... Some wallets/exchanges do not yet support this type of address and may give an error message (e.g. "unknown bitcoin address"). The solution is to manage your funds with a wallet which does support Bech32, <a href="https://en.bitcoin.it/wiki/Bech32_adoption" target="
-                ">see list</a>.
+                            Wasabi generates Bech32 addresses only, also known as bc1 addresses or native SegWit addresses. These addresses start with the characters bc1... Some wallets/exchanges do not yet support this type of address and may give an error message (e.g. "unknown bitcoin address"). The solution is to manage your funds with a wallet which does support Bech32. To check Bech32 adoption and exchange/wallet support you can follow <a href="https://en.bitcoin.it/wiki/Bech32_adoption" rel="external nofollow" target="_blank">Bitcoin.it Wiki</a> and <a href="https://whensegwit.com/#who" rel="external nofollow" target="_blank">When Segwit? website</a>.
```

Comment:
Maybe rephrase it like this:

"Wasabi generates Bech32 addresses only, also known as bc1 addresses or native SegWit addresses. These addresses start with the characters bc1..." from this to "Wasabi generates Bech32 addresses only, also known as native SegWit addresses or bc1 addresses, because these addresses start with the characters bc1..."

> The solution is to manage your funds with a wallet which does support Bech32.

It's not a solution, it's a hack.